ci/helm/values/values-nuxeo-third-party.yaml.gotmpl

```diff
@@ -1,7 +1,4 @@
-{{- $localDomain := printf "%s.svc.cluster.local" .Namespace -}}
-architecture: "api-worker"
-replicaCount: 1
-workerCount: 1
+{{- $localDomain := printf "%s.svc.cluster.local" ( requiredEnv "NAMESPACE" ) -}}
```

Comment:
Also apply in [ci/helm/values/values-nuxeo.yaml.gotmpl](https://github.com/nuxeo/nuxeo-helm-chart/pull/64/files#diff-9df6eff0568e5c00054737a8b8f13ffddf4e9a3a88228f1cc82328dc20aa1d30)?